### DISCUSSION TITLE
I was trying to create experiments using chaos dashboard but I don't see it was getting created

### DISCUSSION THREAD

**Discussion by venkatasivasai1996** [chaos-mesh/chaos-mesh#4326] [Q&A]
## Bug Report

**What version of Kubernetes are you using?**
1.27

**What version of Chaos Mesh are you using?**
v2.6.2

**What did you do? / Minimal Reproducible Example**
trying to create experiment using dashboard but not able to create. I created one using kubectl even though i don't see that in dashboard

**What did you expect to see?**
I would like to create experiment using dashboard and also would like to view experiments created using kubectl

**What did you see instead?**
I wasn't seeing any experiments and events in dashboard

**Output of chaosctl**

**Comment by STRRL** [comment]
could you provide more infomation like browser console log?

**Comment by iflan7744** [comment]
@venkatasivasai1996 Check if you have enabled the debug mode in settings. If so, **disable** it to create an experiment as shown below.

<img width="732" alt="image" src="https://github.com/chaos-mesh/chaos-mesh/assets/55939511/0f567daf-7a2b-4ea3-91d0-30f18bf03160">
